src/main/java/io/jenkins/plugins/checks/github/GitHubChecksDetails.java

```diff
@@ -85,7 +84,7 @@ private AnnotationLevel getAnnotationLevel(final ChecksAnnotationLevel checksLev
             case WARNING:
                 return AnnotationLevel.WARNING;
             default:
-                return null;
+                throw new IllegalArgumentException("unsupported checks annotation level");
```

Comment:
When you throw an exception always add the context. 
```suggestion
                throw new IllegalArgumentException("Unsupported checks annotation level: ", checksLevel);
```

Reply:
Got it